src/reducers.ts

```diff
@@ -51,3 +53,9 @@ function moveShape(state: State, action: MoveShapeAction) {
     })
   });
 }
+
+function bringShapeForward(state: State, action: BringShapeForwardAction) {
```

Comment:
Another point for having these helper function is to have type assertion. What do you recommend to save it? Have 2 functions?